packages/keystone/src/___internal-do-not-use-will-break-in-patch/admin-ui/pages/ListPage/index.tsx

```diff
@@ -742,6 +740,7 @@ const TableHeaderCell = (props: HTMLAttributes<HTMLElement>) => {
         padding: spacing.small,
         textAlign: 'left',
         position: 'sticky',
+        zIndex: 1,
```

Comment:
zIndex manually applied on sticky `thead` element here to maintain the layering order of DOM elements withiin the table, as position: 'relative'  on elements within tbody changes the zindex to be relative to the body